test/modules/internet.spec.ts

```diff
@@ -133,6 +136,15 @@ describe('internet', () => {
           protocol: 'http',
         });
     });
+
+    t.describe('jwt', (t) => {
+      t.it('noArgs', { refDate })
+        .it('with header.alg', { header: { alg: 'ES256' }, refDate })
+        .it('with payload.iss', { payload: { iss: 'Acme' }, refDate })
+        .it('with payload.sub', { payload: { sub: 'subject' }, refDate })
+        .it('with payload.aud', { payload: { aud: 'audience' }, refDate })
+        .it('with payload.jti', { payload: { jti: 'jwt id' }, refDate });
```

Comment:
One is probably enough

```suggestion
        .it('with custom payload', { payload: { iss: 'Acme' }, refDate });
```